common/djangoapps/student/management/commands/change_enrollment.py

```diff
@@ -40,7 +44,8 @@ class Command(BaseCommand):
                     metavar='USER',
                     dest='user',
                     default=False,
-                    help="User to move, if not specified will move all users in the course"),
+                    action="append",
```

Comment:
If you're going with a comma-separated list of users, `action="append"` is unnecessary.